README.md

```diff
@@ -83,7 +83,7 @@ Once we have installed all Khiva dependencies, we are ready to install Khiva fro
 
 - Install conan, c++ package manager, preferably running `pip install conan`. For more information and alternative installation options, please refer to [conan manual page](http://docs.conan.io/en/latest/installation.html).
 - Run `conan remote add conan-mpusz https://api.bintray.com/conan/mpusz/conan-mpusz`.
-- Create `build` folder and, after moving into the new folder, run `conan install .. --build missing`.
+- Create `build` folder and, after moving into the new folder, run `conan install .. --build google-benchmark`.
```

Comment:
Sorry, I did not see this change. Please, maintain "missing" instead of "google-benchmark".

Reply:
Yes, otherwise Google test, boost, etc will not get installed

Reply:
I am sorry for that mistake!

Reply:
No worries! Thanks